rllib/agents/trainer_config.py

```diff
@@ -902,15 +902,23 @@ def offline_data(
                 when the offline file has been generated by another RLlib algorithm
                 (e.g. PPO or SAC), while "normalize_actions" was set to True.
             input_evaluation: DEPRECATED: Use `off_policy_estimation_methods` instead!
-            off_policy_estimation_methods: Specify how to evaluate the current policy.
+            off_policy_estimation_methods: Specify how to evaluate the current policy,
+                along with any optional config parameters.
                 This only has an effect when reading offline experiences
                 ("input" is not "sampler").
-                Available options:
-                - "simulation": Run the environment in the background, but use
+                Available keys:
+                - {"simulation": None}: Run the environment in the background, but use
                 this data for evaluation only and not for learning.
-                - Any subclass of OffPolicyEstimator, e.g.
-                ray.rllib.offline.estimators.is::ImportanceSampling or your own custom
-                subclass.
+                - {ope_method_name: {"type": ope_type, ...}} where `ope_type` can be:
+                    - "is": ImportanceSampling
```

Comment:
That's also part of what I was talking about in the resolved comment further up.
We have the same "issue" at multiple point in our library and I think we should, as you said, allow only full class paths.

But ontop of that provide a helper method that converts config objects between something "yamleable" and objects with actual classes in them. Also because I've ran into issues with providing the StorageUnit enumeration in our config. We can use this method ourselves inside config objects to go back and forth prior to execution time.

Reply:
The problem is I don't think the full classpath works with the `rllib train` utility. I can look at fixing it using an `importlib.import_module`, but I think for now we should keep it, especially since the Offline RL documentation uses that utility.

Actually, I think I can add this fix into the [Offline RL](https://docs.ray.io/en/latest/rllib/rllib-offline.html) docs PR, if that makes sense.

Reply:
I think this is an actual issue, because tune checkpoints will not return said classes and they *must* return full module/class paths for this this model to be sustainable.